test/utils.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  OtokenFactoryInstance,
+  OtokenInstance,
+  MockAddressBookInstance,
+  WhitelistInstance,
+} from '../build/types/truffle-types'
+
+const OTokenFactory = artifacts.require('OtokenFactory.sol')
+const MockAddressBook = artifacts.require('MockAddressBook.sol')
+const Whitelist = artifacts.require('Whitelist.sol')
+const Otoken = artifacts.require('Otoken.sol')
+
+export async function setupContracts(deployer: string) {
+  // Todoo: Use Real addressBook instance when available
+
+  const addressBook: MockAddressBookInstance = await MockAddressBook.new({from: deployer})
```

Comment:
can we use the real instance once it is done? this is a little confusing to keep track of what is mock and what is not